File: samples/java/TicTacToe/Server/src/main/java/systems/zlink/samples/tictactoe/server/configuration/SampleNames.java
package systems.zlink.samples.tictactoe.server.configuration;

import java.time.Duration;

public final class SampleNames {
    public static final String ApiChannel = "tictactoe-api";
    public static final String SpotMesh = "tictactoe";
    public static final String PlayNode = "play";
    public static final String PlayerMilestoneTopic = "tictactoe.player.milestone";
    public static final String PlayStream = "play-stream";
    public static final String PlayActor = "play-actor";
    public static final int RequiredLevel = 3;
    public static final Duration RequestTimeout = Duration.ofSeconds(5);

    private SampleNames() {}
}

File: samples/kotlin/TicTacToe/Server/src/main/kotlin/systems/zlink/samples/kotlin/tictactoe/server/configuration/SampleNames.kt
package systems.zlink.samples.kotlin.tictactoe.server.configuration

import java.time.Duration

object SampleNames {
    const val ApiChannel: String = "tictactoe-api"
    const val SpotMesh: String = "tictactoe"
    const val PlayNode: String = "play"
    const val PlayStream: String = "play-stream"
    const val PlayActor: String = "play-actor"
    const val PlayerMilestoneTopic: String = "tictactoe.player.milestone"
    const val RequiredLevel: Int = 3
    val RequestTimeout: Duration = Duration.ofSeconds(5)
}
